agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -983,7 +983,7 @@ bool ap_manager_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_
 
 void ap_manager_thread::fill_cs_params(beerocks_message::sApChannelSwitch &params)
 {
-    params.tx_power  = ap_wlan_hal->get_radio_info().tx_power;
+    params.tx_power  = (int8_t)ap_wlan_hal->get_radio_info().tx_power;
```

Comment:
```suggestion
    params.tx_power  = static_cast<int8_t>(ap_wlan_hal->get_radio_info().tx_power);
```

Reply:
Actually, for conversions between scalar types, it [should use `T{e}` construction notation](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-construct).
```suggestion
    params.tx_power  = int8_t{ap_wlan_hal->get_radio_info().tx_power};
```

However, this particular one is redundant, because `params.tx_power` is already an int8_t. For the other two it's needed though.